front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx

```diff
@@ -31,9 +56,27 @@ const ScenarioDescription = ({
         </span>
       </div>
 
-      <div className="scenario-description">
+      <div ref={ref} className="scenario-description">
         {scenario.description && (
-          <div className="scenario-details-description">{scenario.description}</div>
+          <div
+            title={!isOpenedDescription ? scenario.description : ''}
+            className={
+              !isOpenedDescription
+                ? 'scenario-details-description not-opened'
+                : 'scenario-details-description opened'
+            }
+            style={!isOpenedDescription ? { width } : { width: width + 120 }}
+          >
+            {scenario.description}
+            <span
+              onClick={() => showDescription()}
+              role="button"
+              tabIndex={0}
+              className={!isOpenedDescription ? 'display-description' : 'displayed-description'}
```

Comment:
Is this better for readability to write it this way ? If it is, you can change it for other rows too

```suggestion
              className={isOpenedDescription ? 'displayed-description' : 'display-description'}
```

Reply:
I agree. Done